lib/client/client.ts

```diff
@@ -103,6 +104,9 @@ export class TypescriptServiceClient {
   executeSaveTo(args: protocol.SavetoRequestArgs) {
     return this.execute("saveto", args)
   }
+  executeNavTree(args: protocol.FileRequestArgs) {
```

Comment:
You should specify the return type here. It's something like `Promise<protocol.NavTreeResponse>`.